### PR TITLE
Convert existing policies to operator policies for installing operators

### PR DESCRIPTION
**Pull request by zyjjay**
Converted policies:
- Compliance operator policy
- Quay container security operator policy
- Gatekeeper operator policy

ref: https://issues.redhat.com/browse/ACM-10573

### CODE REVIEW

**Comment by zyjjay** [comment]
Rebasing then opening it as ready for review.

**Comment by mprahl** [comment]
@zyjjay could you please rebase this PR?

**Comment by zyjjay** [comment]
Seems like `placementrule` and `placementbinding` are deprecated here. I'll go ahead and remove them from these operator policies as well.

File: stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml [hunk 1]
```diff
@@ -32,61 +32,22 @@ spec:
...
+          complianceType: musthave
+          operatorGroup:
+            name: compliance-operator
+            namespace: openshift-compliance
+            targetNamespaces:
+              - openshift-compliance
+          subscription:
+            name: compliance-operator
+            namespace: openshift-compliance
+            installPlanApproval: Automatic
```
Comment (mprahl) [inline stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml]: @zyjjay this is no longer the field to set in the latest release. This was replaced with `spec.upgradeApproval`.

File: stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml [hunk 2]
```diff
@@ -32,61 +32,22 @@ spec:
...
+          complianceType: musthave
+          operatorGroup:
+            name: compliance-operator
+            namespace: openshift-compliance
+            targetNamespaces:
+              - openshift-compliance
+          subscription:
+            name: compliance-operator
+            namespace: openshift-compliance
+            upgradeApproval: Automatic
```
Comment (mprahl) [inline stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml]: upgradeApproval is not at the subscription level. It should be up a level.

Comment (zyjjay) [reply stable/CA-Security-Assessment-and-Authorization/policy-compliance-operator-install.yaml]: Ah good catch :sweat_smile:

**Review by mprahl — APPROVED** [review]

**Pull request merged**